Update issue templates (#13970)
ethereum/solidity: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature Request

> about: Ideas, comments or messages asking for a particular functionality to be added

Reviewer: ```suggestion
about: Ideas and requests for a particular functionality to be added
```

Author: Redundant.

  to Solidity.
title: ''

> labels: Feature Request

Reviewer 2: Do we actually need this label (in general, not just in the template?). We intentionally removed the `feature` label that we had because it was useless. Everything is a feature request or an enhancement by default, unless labeled otherwise, so the label does not add anything. @ekpyron?

Author: I have always used it this way and it helps the submitter to frame either as bugs/feat reqs. Don't see a problem here.

Reviewer 2: I mean, we should all be on the same page regarding this. I don't have anything against having such a label but we should all use it (or not use it) consistently. Given that we decided to remove it, I have no idea when it should be applied and when it should not.

Author: We're going to start needing it soon, so we are doing this.

Reviewer 3: I agree with @cameel, and in fact we configured the triage bot to work on issues that do not contain labels, and thus were not triaged yet. See [here](https://github.com/ethereum/solidity/blob/develop/.github/workflows/triage.yml#L18). If we will change that, we should change the bot action as well.

Reviewer 2: Can we at least stick to the original naming convention? I.e. lower case, words separated with spaces (some other recently introduced labels started to use dashes). We're now using these labels in automation, and when reviewing it's becoming annoying to have to always check which form we're using for a particular label.

If we want this label back, I think we should just restore the original `feature` label.

Author: Totally ok with that.

Reviewer 2: ok then. I renamed this and other labels that could be renamed without changing the code. Pushed #14008 to deal with the ones already referenced in the code.

> labels: Feature Request

Reviewer 2: I renamed the label so this must be changed to match:
```suggestion
labels: feature
```
